amr-wind/physics/multiphase/SloshingTank.H

```diff
@@ -44,6 +45,11 @@ private:
 
     //! Initial zero-level free-surface water depth
     amrex::Real m_waterlevel{0.0};
+
+    //! Stuff to get from MultiPhase physics
+    amrex::Real m_rho1{1000.};
+    amrex::Real m_rho2{1.};
+    amrex::Vector<amrex::Real> m_gravity{0.0, 0.0, -9.81};
```

Comment:
I think we went through gravity usages in the code not too long ago. Can this leverage another, central gravity? To avoid the issues we had with density?

Reply:
in SloshingTank.cpp, this gravity value is obtained from a more central place (MultiPhase physics), so the default isn't used. However, I store it in the class so I don't have to get it more than once. Would it be better to have no initial value?

Reply:
yeah maybe that's better? It would be good to have an obvious fail point?

Reply:
what about initializing with std::numeric_limits<amrex::Real>::quiet_NaN()?